tests/ZendTest/Cache/Storage/Adapter/MongoDbResourceManagerTest.php

```diff
@@ -110,14 +66,8 @@ public function testSetResourceThrowsException()
         $id = 'foo';
```

Comment:
This test fails currently, indicating that a `Zend\Cache\Exception\InvalidArgumentException` was not thrown. When I look at `MongoDbResourceManager::setResource()` there is no validation of the `$resource` argument. The typehint specifies array or `MongoCollection`, but does not test for anything other than `MongoCollection` (in which case it branches logic). I'll update it on merge to raise the exception if `$resource` is neither of the two types.